### PR TITLE
Update Set-up-Hyper-V-Replica.md

### PR DESCRIPTION
**Pull request by paulbaber**
Changed "primary" to "replica" so it now reads, "To run a test failover right-click the replica virtual machine"

### CODE REVIEW

**Comment by PRMerger13** [comment]
@paulbaber : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by carlosmayol — APPROVED** [review]
I agree with the change

File: WindowsServerDocs/virtualization/hyper-v/manage/Set-up-Hyper-V-Replica.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ Do the following on each virtual machine you want to replicate:
...
 
 **Planned failover**: To run a planned failover right-click the primary virtual machine and select **Replication** > **Planned Failover**. Planned failover performs prerequisites checks to ensure zero data loss. It checks that the primary virtual machine is shut down before beginning the failover. After the virtual machine is failed over, it starts replicating the changes back to the primary site when it's available. Note that for this to work the  primary server should be configured to recive replication from the secondary server or from the Hyper-V Replica Broker in the case of a primary cluster. Planned failover sends the last set of tracked changes. [Read more](https://docs.microsoft.com/virtualization/community/team-blog/2012/20120731-types-of-failover-operations-in-hyper-v-replica-part-ii-planned-failover).
```
Comment (illfated) [inline WindowsServerDocs/virtualization/hyper-v/manage/Set-up-Hyper-V-Replica.md:118]: Grammatical detail: Can we also add some commas, to indicate that the sentences actually have got more than one grammatical clause?
```suggestion
**Test failover**:  If you want to run a test failover, right-click the replica virtual machine and select **Replication** > **Test Failover**. Pick the latest or other recovery point if configured. A new test virtual machine will be created and started on the secondary site. After you've finished testing, select  **Stop Test Failover** on the replica virtual machine to clean it up. Note that for a virtual machine you can only run one test failover at a time. [Read more](https://docs.microsoft.com/virtualization/community/team-blog/2012/20120725-types-of-failover-operations-in-hyper-v-replica-part-i-test-failover).

**Planned failover**: To run a planned failover, right-click the primary virtual machine and select **Replication** > **Planned Failover**. Planned failover performs prerequisites checks to ensure zero data loss. It checks that the primary virtual machine is shut down before beginning the failover. After the virtual machine is failed over, it starts replicating the changes back to the primary site when it's available. Note that for this to work, the  primary server should be configured to receive replication from the secondary server, or from the Hyper-V Replica Broker in the case of a primary cluster. Planned failover sends the last set of tracked changes. [Read more](https://docs.microsoft.com/virtualization/community/team-blog/2012/20120731-types-of-failover-operations-in-hyper-v-replica-part-ii-planned-failover).
```
Also, we here get the chance to correct a typo found in the paragraph after the already edited one.

**Comment by PRMerger10** [comment]
@sethmanheim : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@sethmanheim : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**